src/app/shared/components/arbol-cuentas-contables-debito/arbol-cuentas-contables-debito.component.html

```diff
@@ -0,0 +1,33 @@
+<ng-container *ngIf="!data">
+    <div class="d-flex justify-content-center py-3">
+        <div class="spinner-grow text-primary" role="status"></div>
+    </div>
+</ng-container>
+
+<ng-container *ngIf="data">
+    <table [nbTreeGrid]="dataSource" nbSort (sort)="changeSort($event)" equalColumnsWidth>
+
+        <tr nbTreeGridHeaderRow *nbTreeGridHeaderRowDef="allColumns" class="text-center bg-primary text-light"></tr>
+        <tr nbTreeGridRow *nbTreeGridRowDef="let row; columns: allColumns" ngxSelectedRow [myRow]="row"
+            [selectedRow]="selectedTreeRow" (click)="onSelect(row)"></tr>
+
+        <!-- Columna para Arbol -->
+        <ng-container [nbTreeGridColumnDef]="customColumn">
+            <th nbTreeGridHeaderCell [nbSortHeader]="getDirection(customColumn)" *nbTreeGridHeaderCellDef>
+                {{customColumn}}
+            </th>
+            <td nbTreeGridCell *nbTreeGridCellDef="let row">
+                <nb-tree-grid-row-toggle [expanded]="row.expanded" *ngIf="row.children">
+                </nb-tree-grid-row-toggle>
+                {{row.data.Codigo}}
+            </td>
+        </ng-container>
+        <!-- Columnas para Tabla de Descripcion -->
+        <ng-container *ngFor="let column of defaultColumns" [nbTreeGridColumnDef]="column">
+            <th nbTreeGridHeaderCell [nbSortHeader]="getDirection(column)" *nbTreeGridHeaderCellDef>
+                {{column}}
+            </th>
+            <td nbTreeGridCell *nbTreeGridCellDef="let row">{{row.data[column]}}</td>
+        </ng-container>
+    </table>
+</ng-container>
```

Comment:
(En general) Dejar vacía la última línea de cualquier archivo de texto seguido por Git
```suggestion
</ng-container>

```